### ISSUE TITLE
New version release

### ISSUE THREAD

**Issue by sobolevn** [PyCQA/pep8-naming#88]
Hi, thanks for this project! I am using it with great success as a dependency inside my own linter: https://github.com/wemake-services/wemake-python-styleguide

But, there were some blocking issues for me that were already addressed and merged into master.
Like #80 and #84. But these fixed are not yet released.

[The latest release](https://github.com/PyCQA/pep8-naming/releases) was made almost half a year ago. So, it would be awesome to have a new one. Currently I am blocked by it in my own releasing cycle with `wemake-python-styleguide`.

If there's anything I could help with - I would be more than happy to help.
